packages/did-provider-web/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@veramo/core-types": "workspace:^",
     "@veramo/did-manager": "workspace:^",
+    "@veramo/utils": "workspace:^",
```

Comment:
I think we should also have a reference in the `did-provider-web/tsconfig.json` for this workspace package